teuthology/orchestra/run.py

```diff
@@ -87,12 +87,8 @@ def execute(self):
         """
         Execute remote command
         """
-        prefix = "Running:"
-        if self.label:
-            prefix = "Running ({label}):".format(label=self.label)
-        log.getChild(self.hostname).info(prefix)
         for line in self.command.split('\n'):
-            log.getChild(self.hostname).info('> %s' % line)
+            log.getChild(self.hostname).info('%s> %s' % (self.label or '', line))
```

Comment:
Are you sure it is concise duplicating a 'label' for a multi-line script?

Reply:
the label is generally short.  IMO it'll be more readable.  the main thing is getting rid of the Running: line before every command... in reality the label thing is almost never used

Reply:
Ah :-) I found "Running:" useful to quickly jump through the commands, it is pretty opaque to jump with '>'.

Reply:
i'll pull this out into a separate PR since it's unrelated and cosmetic